requirements.txt

```diff
@@ -8,5 +8,6 @@ matplotlib
 numpy
 ophyd
 pymongo
+pypylon
 tfs-pandas
```

Comment:
Let's review the dependencies of this package. I don't think we need `tfs-pandas` in this case. I think we can leave only `ophyd`, `numpy`, and `event-model` for the code itself, and move `bluesky`, `databroker` (and others) to `requirements-dev.txt` - for testing purposes only. We'd like to make sure we have a relatively lightweight package.